apps/projects/templates/a4_candy_projects/includes/project_list_tile.html

```diff
@@ -28,9 +28,9 @@
         </div>
     </div>
     <div class="tile__body tile__body-status">
-        <h3 class="tile__title" id="project-title-{{ project.pk }}">
+        <h4 class="tile__title" id="project-title-{{ project.pk }}">
             <a href="{{ project|project_url }}">{{ project.name }}</a>
-        </h3>
+        </h4>
```

Comment:
for a11y we shouldn't actually change the tag as it is the third heading on the page, we can change the size with css though

Reply:
yes i checked it and the title of this section is already `h3`, so i thought it would make sense as these are nested. But i don't know if `h2` or `h1` are used though

Reply:
@khamui it looks like we messed up with that, I think we should have 'following' as h2 and maybe have a hidden h1 saying 'user overview' or something but maybe for now just sort out the h2 and h3?